keyboards/keyhive/maypad/maypad.h

```diff
@@ -0,0 +1,49 @@
+/* Copyright 2019 codybender
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "quantum.h"
+
+/* maypad numpad layout
+ * .-------------------.
+ * |NLCK|   /|   *|   -|
+ * |-------------------|
+ * |   7|   8|   9|    |
+ * |--------------|    |
+ * |   4|   5|   6|   +|
+ * |-------------------|
+ * |   1|   2|   3|    |
+ * |--------------|    |
+ * |        0|   .| Ent|
+ * '-------------------'
+ */
+// The first section contains all of the arguments
+// The second converts the arguments into a two-dimensional array
+#define LAYOUT_numpad_5x4( \
```

Comment:
Where did the `ortho_5x4` layout go? :( You should add it here and in info.json, as this board supports both.

Reply:
I removed it for simplicity, since I really had no idea what was going on. is it just a second define statement using `ortho_5x4`? And then the corresponding changes in the keymap

Reply:
Basically, if someone builds a Maypad with, say, two 1u keys instead of the normal Numpad +, this gives them a macro that they can use in their keymap to access those positions on the PCB.

Reply:
Yep, but you don't need to do anything in the default keymap.c - it's so that people who aren't using the numpad layout can just use `LAYOUT_ortho_5x4(...)` in their keymaps. The keyboard should declare all possible layouts (within reason, some keyboards support *a lot* of configurations) so you can pick the one that matches how you've placed the switches.